include/tunnler/room_member.h

```diff
@@ -86,13 +90,13 @@ class RoomMember final {
      * Sends a chat message to the room.
      * @param message The contents of the message.
      */
-    void SendChatMessage(const std::string message);
+    void SendChatMessage(const std::string& message);
 
 	/**
      * Sends a WiFi packet to the room.
      * @param packet The WiFi packet to send.
      */
-    void SendWifiPacket(const WifiPacket& packet);
+    void SendWifiPacket(WifiPacket& packet);
```

Comment:
To resolve this before the merge: please make it accept a const ref. I feel like this is part of a stable API so it would suck to change it later